webapp/src/main/resources/public/js/components/workbench/GitBlameInfoModal.js

```diff
@@ -43,6 +44,28 @@ let GitBlameInfoModal = React.createClass({
         );
     },
 
+    /**
+     * @param label                 The label for the data to display
+     * @param data                  The link text
+     * @param branchScreenshots     The screenshots for this branch
+     * @returns {*}                 The row of label:link to display in the modal
+     */
+    displayScreenshotLink(label, data, branchScreenshots) {
+        if (data == null || !branchScreenshots || !branchScreenshots.length) {
+            return this.displayInfo(label,null)
+        }
+        return (
+            <div className={"row git-blame"}>
+                <label className={"col-sm-3 git-blame-label"}>{label}</label>
+                <Link
+                    onClick={() => GitBlameScreenshotViewerActions.open(branchScreenshots)}
```

Comment:
would you mind having a prop for the "onClick" callback? 

Trying to keep the UI component "dumb" and let the top component do all the wiring